smartcontract/common/common.go

```diff
@@ -95,3 +97,184 @@ func convertNeoVmTypeHexString(item interface{}, count *int) (interface{}, error
 		return nil, errors.New("[ConvertTypes] Invalid Types!")
 	}
 }
+
+func Stringify(item interface{}) (string, error) {
+	var count int
+	return stringify(item, &count)
+}
+func stringify(item interface{}, count *int) (string, error) {
+	if item == nil {
+		return "", nil
+	}
+	if *count > MAX_COUNT {
+		return "", errors.New("over max parameters convert length")
+	}
+	switch v := item.(type) {
+	case *types.Boolean:
+		b, err := v.GetByteArray()
+		if err != nil {
+			return "", err
+		}
+		return fmt.Sprintf("bytes(hex:%x)", b), nil
+	case *types.ByteArray:
+		bs, err := v.GetByteArray()
+		if err != nil {
+			return "", err
+		}
+		return fmt.Sprintf("bytes(hex:%x)", bs), nil
+	case *types.Integer:
```

Comment:
use `case *types.Boolean, *types.BytesArray, *types.Integer:` to avoid duplicated code